### DISCUSSION TITLE
Kdialog errors (firecfg usage issue)

### DISCUSSION THREAD

**Discussion by fjnewt** [netblue30/firejail#4942] [Q&A] [answered]
I wonder if I can get some help here. I'm using KDE5 on PCLOS,  re Firejail 0.9.68 and Firetools 0.9.64.

I run a few apps from FJ. I was never able to use the Add Books or Save functions in Calibre via FJ neither can I access files from Gweview when run from FJ. I get KDialog errors for both: 
Unable to create io-slave. Can not create socket for launching io-slave for protocol 'file'. and 
Unable to create io-slave. Can not create socket for launching io-slave for protocol 'tags'.

I checked FJ's version via CL and got this error as well.
Error: cannot read UID_MIN and/or GID_MIN from /etc/login.defs, using 1000 by default
firejail version 0.9.68

Other apps like Thunderbird, GIMP, Inkscape FF, Seamonkey run without issues (that I know of)

Can anyone shed any light on what could be happening. 

I must add that I ran  'su firecfg' and then had to add my username in the /etc/firejail/firejail.users file, as none of the apps would run in FJ.

Thanks for any comment/suggestion.

**Comment by rusty-snake** [comment, marked as answer]
> I ran 'su firecfg' 

Probably better to use `sudo`.
